fittotcx/program.py

```diff
@@ -234,12 +234,17 @@ def main():
             description="This program takes a FIT file and converts it " +
                         "into an TCX file and output the result to the " +
                         "standard output.")
-    parser.add_argument('file', metavar='FILE', type=argparse.FileType('r'))
+    parser.add_argument('inputfile', metavar='INPUTFILE', nargs=1, type=str)
+    parser.add_argument('outputfile', metavar='OUTPUTFILE', nargs='?', type=str, default=None)
```

Comment:
This should be `type=argparse.FileType('w'), default=sys.stdout`, which will allow a substantial simplification below (as well as error checking for unwriteable files by `argparse`).